projects/storefrontlib/src/lib/checkout/services/checkout-details.service.ts

```diff
@@ -26,23 +17,15 @@ export class CheckoutDetailsService {
   getCheckoutDetailsLoaded$: Observable<boolean>;
```

Comment:
I can't add a comment on that line but like 24, is the `userId$` used anywhere or can it be removed?

Reply:
Good catch!

Fixed.